src-backend/api/tests/test_model_page.py

```diff
@@ -88,3 +89,20 @@ def test_page_ordering(self):
 
         for i in range(0, 4):
             assert_equals(pages[i], expected_pages[i])
+
+    def test_page_reordering(self):
+        page1 = Page.objects.create(
+            display_index=0,
+            procedure=self.test_procedure1
+        )
+
+        page2 = Page.objects.create(
+            display_index=0,
```

Comment:
Maybe give this one a `display_index` of 1? Would be more realistic since we'll have integrity constraints anyway.